Dockerfile

```diff
@@ -23,6 +27,17 @@ RUN apt-get install -y cmake
 
 WORKDIR /scripts
 
+# Install R dependency packages
+RUN R -e "install.packages('arrow')"
+RUN R -e "install.packages('data.table')"
+RUN R -e "install.packages('doParallel')"
+RUN R -e "install.packages('foreach')"
```

Comment:
Did moving the order of R vs python resolve the issues?

Reply:
No, however it's a quality of life change. The Docker builds the Dockerfile line by line, and we do not expect the R packages to be changed very often if at all. If Docker does not notice changes based on a line, then it'll skip it. However the Python stuff will always be changing, so it'll cut development build time.